### DISCUSSION TITLE
[HSR-C] Problem of delayed initial movement of arm joint

### DISCUSSION THREAD

**Discussion by tsunotaro** [HumanSupportRobot/QandA#107] [Q&A]
Hello.

I am currently using HSRC.
The problem I encountered was that the initial movement of the arm joint was slow, even after "time synchronization".

I perform time synchronization using the method described in HSRC user manual, "6.2.3.1.2 If the time synchronization server is not available”

I have seen that repeating the "time synchronization" multiple times can sometimes solve the problem, but I'm not sure if that's the right way to do it.

If anyone knows the correct way or an alternative way, I would appreciate any information or references you can provide.

Thank you.

**Comment by yutaro-ishida** [comment]
To determine whether or not time synchronization is the cause, why not use `$ chronyc sources` or `$ chronyc tracking` to be sure that time synchronization is working correctly when the problem occurs?

**Reply by tsunotaro** [reply]
Thank you for your reply.

Also, sorry for the late reply.

When the problem occurred, I confirmed that the time sync was working properly using `$ chronyc source` and `$ chronyc tracking`.

In the end, the cause of this problem was the Wi-Fi I was using.
We are currently investigating why the problem occurs when using a specific Wi-Fi.

Thank you for the information you provided.
